src/css/cs/items/_kitchensink.scss

```diff
@@ -347,6 +347,10 @@
     }
   }
 
+  .uc-white-space-pre-wrap {
+    white-space: pre-wrap;
```

Comment:
@marcpechaitis fine with me, in CalCentral we usually use `pre-line`

https://github.com/ets-berkeley-edu/calcentral/blob/73adef1233c2fbd565d4cbc17d6a9e44a30c7b64/src/assets/stylesheets/_text.scss#L21